litgpt/chat/base.py

```diff
@@ -120,6 +165,7 @@ def main(
     quantize: Optional[Literal["bnb.nf4", "bnb.nf4-dq", "bnb.fp4", "bnb.fp4-dq", "bnb.int8"]] = None,
     precision: Optional[str] = None,
     compile: bool = False,
+    multiline_prompts: bool = False,
```

Comment:
What else could be multiline?

```suggestion
    multiline: bool = False,
```

Reply:
agreed, let's simplify